### PR TITLE
Update Developer Guide with implementation of PriorityP command

### LINKED ISSUE #125: Update implementation of PriorityP command in Developer Guide

### PR DESCRIPTION
**Pull request by rayyan35p**
Closes #125

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -172,6 +172,32 @@ Step 5. During the execution of the `AddProjectCommand`, selected `Employee` obj
 
 Step 6. A `CommandResult` is produced based on whether the execution was a success or not and returned to the `LogicManager`. 
 
+### Prioritising Projects Feature
```
Comment (aslam341) [inline docs/DeveloperGuide.md:175]: `Prioritise` would be better than `Prioritising`

File: docs/diagrams/PriorityPSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":TaskHubParser" as TaskHubParser LOGIC_COLOR
+participant "p:PriorityProjectCommand" as PriorityProjectCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+[-> LogicManager : execute(priorityP 1 priority/high)
```
Comment (aslam341) [inline docs/diagrams/PriorityPSequenceDiagram.puml:14]: Argument in execute should be in double quotes `" "`.

File: docs/diagrams/PriorityPSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+activate LogicManager
+
+LogicManager -> TaskHubParser : parseCommand(priorityP 1 priority/high)
```
Comment (aslam341) [inline docs/diagrams/PriorityPSequenceDiagram.puml:17]: Argument in parseCommand should be in double quotes `" "`.

**Review by aslam341 — CHANGES_REQUESTED** [review]
LGTM, just some minor standardizing issues.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (rayyan35p) [reply docs/DeveloperGuide.md:175]: Sure!

File: docs/diagrams/PriorityPSequenceDiagram.puml (hunk 2 above)
Comment (rayyan35p) [reply docs/diagrams/PriorityPSequenceDiagram.puml:14]: Noted!

File: docs/diagrams/PriorityPSequenceDiagram.puml (hunk 3 above)
Comment (rayyan35p) [reply docs/diagrams/PriorityPSequenceDiagram.puml:17]: Noted on this as well, thanks!

**Review by aslam341 — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -172,6 +172,32 @@ Step 5. During the execution of the `AddProjectCommand`, selected `Employee` obj
 
 Step 6. A `CommandResult` is produced based on whether the execution was a success or not and returned to the `LogicManager`. 
 
+### Prioritise Projects Feature
+
+Upon creation of a new `Project`, a `ProjectPriority` can be assigned to that `Project` using the `PriorityP` command. 
+
```
Comment (Chandan8186) [inline docs/DeveloperGuide.md:178]: Perhaps it should be `priorityP` command instead of `PriorityP`?

**Review by Chandan8186 — COMMENTED** [review]
LGTM! Just a minor change for the feature description.

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (rayyan35p) [reply docs/DeveloperGuide.md:178]: Yep, good catch. I will try to fix that soon in another PR.

Comment (aslam341) [reply docs/DeveloperGuide.md:178]: @Chandan8186 I have changed this in PR #130.

**Pull request merged**